simcore/simulation/library/anchor.hpp

```diff
@@ -16,7 +16,8 @@ class Anchor : public Object {
   bool end_pausing_;
   crosslink_parameters *sparams_;
   int step_direction_;
-
+   
+  double kick_amplitude;
```

Comment:
When defining member variables in c++, add underscore afterwards. This distinguishes local from member variables in methods.